src/realizations/catchment/CMakeLists.txt

```diff
@@ -29,7 +29,6 @@ target_link_libraries(realizations_catchment PUBLIC
         )
 
 if(NGEN_ACTIVATE_PYTHON)
-   target_include_directories(realizations_catchment PUBLIC ${PROJECT_SOURCE_DIR}/extern/pybind11/include)
```

Comment:
This change was something I noticed as I was implementing the main goal of this PR